odxtools/templates/macros/printResponse.xml.jinja2

```diff
@@ -10,7 +10,9 @@
 {%- macro printResponse(resp, tag_name="POS-RESPONSE") -%}
 <{{tag_name}} ID="{{resp.odx_id.local_id}}">
  <SHORT-NAME>{{resp.short_name}}</SHORT-NAME>
+{%- if resp.long_name %}
  <LONG-NAME>{{resp.long_name|e}}</LONG-NAME>
+{%- endif %}
```

Comment:
Drive by fix?

Reply:
yes. That one was caused because for some reason, with the other changes included in this PR, my test script started to fail because a internal file which was passed through [`examples/pdxcopy.py`](https://github.com/mercedes-benz/odxtools/blob/main/examples/pdxcopy.py) started produce different results than the original for the `odxtools list` subcommand. there are probably a few more of these (sorry for that)...